macros/schema_tests/cardinality_equality.sql

```diff
@@ -0,0 +1,28 @@
+{% macro test_cardinality_equality(model, from, to, field) %}
```

Comment:
@dwallace0723 the `from`, `to`, and `field` args can be named arbitrarily -- we used these for the referential integrity test, but you can pick other names if you'd like!

Reply:
I'll probably just leave them as is. The current naming makes logical sense to me. LMK if you think there are more suitable arg names for this specific test.